message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -918,6 +918,84 @@
         <description>Stalling and steering towards the land point</description>
       </entry>
     </enum>
+    <enum name="PLANE_MODE">
+      <description>A mapping of plane flight modes for custom_mode field of heartbeat</description>
+      <entry value="0"  name="PLANE_MODE_MANUAL"/>
+      <entry value="1"  name="PLANE_MODE_CIRCLE"/>
+      <entry value="2"  name="PLANE_MODE_STABILIZE"/>
+      <entry value="3"  name="PLANE_MODE_TRAINING"/>
+      <entry value="4"  name="PLANE_MODE_ACRO"/>
+      <entry value="5"  name="PLANE_MODE_FLY_BY_WIRE_A"/>
+      <entry value="6"  name="PLANE_MODE_FLY_BY_WIRE_B"/>
+      <entry value="7"  name="PLANE_MODE_CRUISE"/>
+      <entry value="8"  name="PLANE_MODE_AUTOTUNE"/>
+      <entry value="10" name="PLANE_MODE_AUTO"/>
+      <entry value="11" name="PLANE_MODE_RTL"/>
+      <entry value="12" name="PLANE_MODE_LOITER"/>
+      <entry value="14" name="PLANE_MODE_AVOID_ADSB"/>
+      <entry value="15" name="PLANE_MODE_GUIDED"/>
+      <entry value="16" name="PLANE_MODE_INITIALISING"/>
```

Comment:
Shold be `INITIALIZING` for consistency.

Reply:
Fixed.